engines/rados.c

```diff
@@ -0,0 +1,501 @@
+/*
+ *  Ceph Rados engine
+ *
+ * IO engine using Ceph's RADOS interface to test low-level performance of
+ * Ceph OSDs.
+ *
+ */
+
+#include <rados/librados.h>
+
+#include "fio.h"
+#include "../optgroup.h"
+
+struct fio_rados_iou {
+	struct io_u *io_u;
+	rados_completion_t completion;
+	rados_write_op_t write_op;
+	int io_seen;
+	int io_complete;
+};
+
+struct rados_data {
+	rados_t cluster;
+	rados_ioctx_t io_ctx;
+	char** objects;
```

Comment:
Please follow the fio style, which would make this:

```
char **objects;
```
like you actually do for other members.